clusterInfo.py

```diff
@@ -8,6 +7,9 @@
 
 ENV_CDA_CLUSTERINFO_CREDENTIALS = "CDA_CLUSTERINFO_CREDENTIALS"
 
+SHEET = "ANL lab HW enablement clusters and connections"
+URL = "https://docs.google.com/spreadsheets/d/1lXvcodJ8dmc_hcp0hzbPDU8t6-hCnAlEWFRdM2r_n0Q"
```

Comment:
Is there a reason to make this url publicly available? This is the first that this url apprears in the public.

Reply:
I would not call this "publicly available".

The URL was hard-coded in the sources already before. All it does, is move this to a separate variable.

This variable is actually used in later commits (so that `python clusterInfo.py --help` can print it).

Reply:
> The URL was hard-coded in the sources already before.

Ah. It wasn't. I didn't realize that.

It seems useful that the logging of CDA and `python clusterInfo.py --help` shows the URL and I can click on it. Please read the output of `python clusterInfo.py --help`.

I don't think this URL is private sensitive information. If you don't have credentials, you only see a cryptic URL. The name of the sheet already reveals more information.

Granted, without the URL, a external user could add their own  "ANL lab HW enablement clusters and connections" sheet and read that. However, nobody is going to do that, because the use is obscure and undocumented.

Reply:
@bn222 is it then OK to have the URL in code? (and shown in logs and `--help` output)?